appmock/src/main/java/space/taran/arknavigator/mvp/model/repo/index/ResourceId.kt

```diff
@@ -1,11 +1,9 @@
 package space.taran.arknavigator.mvp.model.repo.index
 
+import space.taran.arklib.ResourceId
+import space.taran.arklib.computeId
 import java.nio.file.Path
 
-typealias ResourceId = Long
-
-private external fun computeIdNative(size: Long, file: String): Long
-
-fun computeId(size: Long, file: Path): ResourceId {
-    return computeIdNative(size, file.toString())
+fun computeID(size: Long, file: Path): ResourceId {
+    return computeId(size, file)
```

Comment:
`computeId` is needed in the library module(since we are publishing it), not in the `appmock` module
btw `appmock` is a bad name, it's more like `sample`. Can you rename it?

Reply:
ah, you have already added this function to the library module

Then remove `computeID` from `appmock` and rename this module to `sample`